docs/source/configuration/persisted-queries.mdx

```diff
@@ -1,72 +1,178 @@
 
 ```yaml title="router.yaml"
 preview_persisted_queries:
   enabled: true
   log_unknown: true
+apq:
+  enabled: false # APQ must be disabled
 ```
 
-### Enabling safelisting
+Unregistered operations appear in your [router's logs](./logging).
+
+#### Safelisting (operation strings allowed)
 
-Enabling safelisting requires creation of a Persisted Query List (PQL). Refer to the [Persisted Query documentation](https://www.apollographql.com/docs/graphos/operations/persisted-queries) for more information on how to create these and update your clients. 
+> Before applying this configuration, ensure your PQL contains _all_ of the GraphQL operations that _all active versions_ of your clients execute. If you enable safelisting _without_ ensuring this, your router will reject any unpublished client operations.
 
-Once you have a PQL set up then you can enable safelisting in your router's [YAML config file](./overview/#yaml-config-file), like so:
+With the following configuration, your router allows _only_ GraphQL operations that are present in its PQL while rejecting all other operations:
 
 ```yaml title="router.yaml"
 preview_persisted_queries:
   enabled: true
+  log_unknown: true
   safelist:
     enabled: true
+    require_id: false
+apq:
+  enabled: false # APQ must be disabled 
 ```
 
-### Requiring IDs
+To execute an operation, clients can provide its full operation string _or_ its PQL-specified ID.
+
+#### Safelisting (IDs only)
 
-Persisted queries uses a similar mechanism to APQ to allow clients to send queries as IDs and thus omit the body of the graphql query. This can significantly reduce the size of the request body and improve performance.
+> **Do not start with this configuration:** It requires _all_ your clients execute operations by providing their PQL-specified ID. If any clients still provide full operation strings, those clients will break.
+
+With the following configuration, your router allows _only_ GraphQL operations that are present in its PQL, _and_ it requires clients to provide PQL-specified IDs instead of full operation strings:
 
 ```yaml title="router.yaml"
 preview_persisted_queries:
   enabled: true
+  log_unknown: true
   safelist:
     enabled: true
-    require_id: true
+    require_id: true #highlight-line
+apq:
+  enabled: false # APQ must be disabled 
+```
+
+If you want to use this security level, you should always _first_ set up [safelisting with operation strings allowed](#safelisting-operation-strings-allowed). This level requires modifying _all_ of your clients to execute operations via PQL-specified ID instead of an operation string. While making those necessary changes, you can use the less restrictive safelisting level in your router.
+
+## Router logs
+
+If a router instance is set to [audit mode](#audit-dry-run), its [logs](./logging) will include any operations it recieves that haven't been added to the PQL.
+
+For example:
+
 ```
+To-do
```

Comment:
Could you please add an example of how unregistered operations will appear in the router logs?